src/core/libultra/os.cpp

```diff
@@ -37,8 +37,8 @@ int32_t osContStartReadData(OSMesgQueue* mesg) {
 void osContGetReadData(OSContPad* pad) {
     memset(pad, 0, sizeof(OSContPad) * __osMaxControllers);
 
-    auto inputEditor = LUS::Context::GetInstance()->GetWindow()->GetGui()->GetGuiWindow("Input Editor");
-    if (!inputEditor || !inputEditor->IsVisible()) {
+    auto inputEditor = std::reinterpret_pointer_cast<LUS::InputEditorWindow>(LUS::Context::GetInstance()->GetWindow()->GetGui()->GetGuiWindow("Input Editor"));
+    if (!inputEditor || inputEditor->ShouldBlockGameInput()) {
```

Comment:
if we just want to get old functionality back all we need to do is change this one line to be
```cpp
if (!inputEditor || inputEditor->IsVisible()) {
```
so we don't block input when the controller config menu is hidden but instead block it when it's visible (which is how it works in spock)